src/main/webapp/Portal.html

```diff
@@ -121,7 +121,7 @@
 			head.load(cdnEndpoint + "generated/jquery.min.js");
 			head.load("//ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js");
 			head.load(cdnEndpoint + "js/back-forward-nav-handler.js");
-			head.load(cdnEndpoint + "js/moment.min.js");
+			head.load(cdnEndpoint + "generated/moment.min.js");
```

Comment:
Load the moment.min.js that was referenced in package.json (instead of the static copy)